### PR TITLE
testnet-deployment

### PR DESCRIPTION
**Pull request by kaisbaccour**

### CODE REVIEW

File: script/util/copy-verifier.sh [hunk 1]
```diff
@@ -20,6 +20,9 @@ case "$ENV" in
   dev-3)
     VERIFIER_SOL_URL="https://pub-bca6985bd0e849b5b8840edc0b7f9e15.r2.dev"
     ;;
+  holesky)
+    VERIFIER_SOL_URL="https://pub-fbb5db8dc9ee4e8da9daf13e07d27c24.r2.dev"
+    ;;
   base_sepolia | fraxtal_testnet | holesky | scroll_sepolia)
     VERIFIER_SOL_URL="https://pub-fbb5db8dc9ee4e8da9daf13e07d27c24.r2.dev"
     ;;
```
Comment (RyanRHall) [inline script/util/copy-verifier.sh:28]: I think this case is unnecessary because `holesky` is included in the case on line 26, right?

**Review by nikkolasg — APPROVED** [review]

**Review by RyanRHall — CHANGES_REQUESTED** [review]

File: script/util/copy-verifier.sh (hunk 1 above)
Comment (kaisbaccour) [reply script/util/copy-verifier.sh:28]: oh yeah I havent seen this. makes sense

Comment (kaisbaccour) [reply script/util/copy-verifier.sh]: fixed it

**Review by RyanRHall — APPROVED** [review]
LGTM 👍

**Pull request merged**